src/Business Foundation/Test/NoSeriesCopilot/src/NoSeriesCopilotHarmsTests.Codeunit.al

```diff
@@ -29,8 +29,17 @@ codeunit 134541 "No. Series Copilot Harms Tests"
         TestInputJson: Codeunit "Test Input Json";
     begin
         TestInputJson := AITTestContext.GetQuestion();
-        asserterror NoSeriesCopilotTestLib.Generate(NoSeriesGeneration, NoSeriesGenerationDetail, TestInputJson.ValueAsText());
+        if not CallGenerateFunction(NoSeriesGeneration, NoSeriesGenerationDetail, TestInputJson.ValueAsText()) then begin
+            assert.ExpectedError(InvalidPromptTxt);
```

Comment:
```suggestion
            Assert.ExpectedError(InvalidPromptTxt);
```

Reply:
Will address in next update. One more to follow real soon 😊